public/app/features/alerting/unified/components/Authorize.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, ReactNode } from 'react';
+import { AccessControlAction } from 'app/types';
+import { contextSrv } from 'app/core/services/context_srv';
+
+type Props = {
+  actions: AccessControlAction[];
+  fallback?: boolean;
+  children: ReactNode;
```

Comment:
I think this is redundant, `FC` is automatically typed as `PropsWithChildren`

Reply:
TIL 😸